.gitignore

```diff
@@ -6,6 +6,7 @@
 *.user
 *.userosscache
 *.sln.docstates
+*.launchSettings.json 
```

Comment:
This should just `launchSettings.json`. If you put `*.` before the name will only match files called `something.launchSettings.json`. Also you will need to ask Git to stop tracking `launchSettings.json` because it's already tracking it: `git rm --cached YoCode\Properties\launchSettings.json`